Dockerfile

```diff
@@ -1,13 +1,23 @@
 #############      builder       #############
-FROM golang:1.14.2 AS builder
+FROM golang:1.15.7 AS builder
```

Comment:
There is some other places where golang image is also used, e.g. https://github.com/gardener/logging/blob/c4a3498932b499bafdba17d3bf41e15be0f00de5/.ci/pipeline_definitions#L23